tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -467,21 +467,35 @@ def test_show_queue_counters(self, setup, setup_config_mode, duthosts, enum_rand
 
         for key in buffer_queue_keys:
             try:
-                interfaces.add(key.split('|')[1])
+                # The interface name is always the last but one field in the BUFFER_QUEUE entry key
+                interfaces.add(key.split('|')[-2])
             except IndexError:
                 pass
 
+        # For the test to be valid, we should have at least one interface selected
+        assert (len(interfaces) > 0)
+
+        intfsChecked = 0
         if mode == 'alias':
             for intf in interfaces:
-                alias = setup['port_name_map'][intf]
+                alias = setup['port_name_map'].get(intf)
+                if not alias:
+                    continue
                 assert (re.search(r'{}\s+[U|M]C|ALL\d\s+\S+\s+\S+\s+\S+\s+\S+'
                                   .format(alias), queue_counter) is not None) \
                     and (setup['port_alias_map'][alias] not in queue_counter)
+                intfsChecked += 1
         elif mode == 'default':
             for intf in interfaces:
+                if intf not in setup['port_name_map']:
+                    continue
```

Comment:
please make this change specific to voq chassis

Reply:
Fixed in latest.

Reply:
don't see any change for this code block